Dockerfile

```diff
@@ -78,14 +78,14 @@ RUN yum -y install --setopt=tsflags=nodocs ruby-install make
 # Install Ruby 2.3.1
 RUN ruby-install ruby 2.3.1 -- --disable-install-doc && rm -rf /usr/local/src/* && yum clean all
 
-## Build Auth-Api
-ENV HTTPD_AUTH_API_SERVICE_DIRECTORY=/opt/auth-api
-RUN mkdir -p ${HTTPD_AUTH_API_SERVICE_DIRECTORY}
-COPY docker-assets/auth-api ${HTTPD_AUTH_API_SERVICE_DIRECTORY}
-RUN  cd ${HTTPD_AUTH_API_SERVICE_DIRECTORY} && \
-     gem install bundler && \
-     bundle install
-COPY docker-assets/auth-api.service    /usr/lib/systemd/system/auth-api.service
+## Install DBus API Service
+ENV HTTPD_DBUS_API_SERVICE_DIRECTORY=/opt/dbus-api-service
```

Comment:
The gem is underscored, and this is dashed...I think we should be consistent and use underscore.